errors/index.js

```diff
@@ -0,0 +1,16 @@
+exports.handleCustomErrors = (err, req, res, next) => {
+    if (err.status && err.msg) {
+        res.status(err.status).send({ msg: err.msg });
+    } else next(err);
+};
+
+exports.handlePsqlErrors = (err, req, res, next) => {
```

Comment:
You don't need this yet, and we're trying to only use enough code to satisfy the conditions of the task so take out your PSQL error handler for now - the handleCustom errors and handleServerErrors are okay :)